File: test/unit/libsrcml/xpath_test_data/is_member/is_member.cpp
class Foo {
    class Foo1          /* 1 */;
    struct Foo2         /* 2 */;
    Foo()               /* 3 */;
    Foo(X const& v)     /* 4 */ { }
    ~Foo();             /* 5 */
    ~Foo()              /* 6 */ { }
    void foo7()         /* 7 */ { }
    void foo8()         /* 8 */;
    class Foo9          /* 9 */  { };
    struct Foo10        /* 10 */ { };
    union Foo11         /* 11 */ { };
    enum Foo12          /* 12 */ { };
    enum class Foo13    /* 13 */ { };
    enum class Foo14    /* 14 */;
    enum Foo15          /* 15 */;
    typedef type Foo16  /* 16 */;
    using type = Foo17  /* 17 */;
    int a               /* 18 & 19 */; 
}

class Foo1;
struct Foo2;
void foo7() { }
void foo8();

enum foo {
    a                   /* 20 */,
    b                   /* 21 */,
};

File: test/unit/libsrcml/xpath_test_data/is_member/is_member.cs
class Foo {
    class Foo1;     /* 1 */
    struct Foo2;    /* 2 */
    Foo();          /* 3 */
    Foo() { }       /* 4 */
    ~Foo();         /* 5 */
    ~Foo() { }      /* 6 */
    void foo7() { } /* 7 */
    void foo8();    /* 8 */
    string foo9 {   /* 9 */
        get         /* 10 */;
        set         /* 11 */;
    }
    event EventType foo12; /* 12 */
    interface Foo14 /* 13 */{ } 
    class Foo15     /* 14 */{ } 
    struct Foo16    /* 15 */{ } 
}

class Foo1;
struct Foo2;
void foo7() { }
void foo8();
string foo9 {
    get;
    set;
}
event EventType foo12;
